web/cat/routes/upload.py

```diff
@@ -15,11 +14,6 @@ async def rabbithole_upload_endpoint(
     request: Request,
     file: UploadFile,
     background_tasks: BackgroundTasks,
-    chunk_size: int = Body(
```

Comment:
Add back `chunk_size` and `chunk_overlap` params to the endpoint, so scripts uploading files to the Cat can decide how the text is splitted and overlapped